client/src/pages/people/Form.js

```diff
@@ -363,11 +366,17 @@ export default class PersonForm extends ValidatableFormWrapper {
 				case 'leftVacant':
 				case 'hasReplacement':
 					// reset account?
-					if (confirm('Are you sure you want to reset this account?')) {
-						let { person } = this.state
-						person.status = 'INACTIVE'
-						this.updatePerson(person, true, optionValue === 'needNewAccount')
-					}
+					confirmAlert({
```

Comment:
@gjvoosten please review. Any idea on how to maintain an consistent style?

Reply:
@vassil Please note: if a super user resets an account, the `confirmLabel` is not correct (it's only valid for the `needNewAccount` case).

Reply:
@vassil I've updated the `confirmLabel`. As to styling: the defaults are not so bad. We could make the button backgrounds blue if you wish. Too bad there's no different style classes for the cancel and confirm buttons, so having the cancel one black text on a white background and the confirm one white text on a blue background would have to depend on the order of the buttons.